SoundSwitch/UI/Forms/Settings.cs

```diff
@@ -824,5 +846,77 @@ private void UsePrimaryScreenCheckbox_CheckedChanged(object sender, EventArgs e)
         }
 
         #endregion
+
+        #region Troubleshooting
+
+        private void ResetAudioDevicesButton_Click(object sender, EventArgs e)
+        {
+            AudioSwitcher.Instance.ResetProcessDeviceConfiguration();
+        }
+
+        private void ExportLogFilesButton_Click(object sender, EventArgs e)
+        {
+            var saveFileDialog = new SaveFileDialog
+            {
+                InitialDirectory = "C://",
+                Title = "Export Log Files",
+                FileName = "soundswitch_logs",
+                DefaultExt = "zip",
+                Filter = "Zip Archive (*.zip)|*.zip",
+                RestoreDirectory = true
+            };
+
+            if (saveFileDialog.ShowDialog() == DialogResult.OK)
+            {
+                if (File.Exists(saveFileDialog.FileName))
+                    File.Delete(saveFileDialog.FileName);
+
+                Log.CloseAndFlush();
+
+                var files = Directory.EnumerateFiles(ApplicationPath.Logs, "*.log");
+                var zip = ZipFile.Open(saveFileDialog.FileName, ZipArchiveMode.Create);
+                foreach (var file in files)
+                {
+                    // Add the entry for each file
+                    zip.CreateEntryFromFile(file, Path.GetFileName(file), CompressionLevel.Optimal);
+                }
+                // Dispose of the object when we are done
+                zip.Dispose();
```

Comment:
```suggestion
                using var zip = ZipFile.Open(saveFileDialog.FileName, ZipArchiveMode.Create);
                foreach (var file in files)
                {
                    // Add the entry for each file
                    zip.CreateEntryFromFile(file, Path.GetFileName(file), CompressionLevel.Optimal);
                }
```

By utilizing the `using` keyword you don't need to manually dispose. It'll be done at the end of the function.